projects/packages/admin-ui/src/class-admin-menu.php

```diff
@@ -5,224 +5,152 @@
  * @package automattic/jetpack-admin-ui
  */
 
+declare(strict_types=1);
```

Comment:
We haven't used strict typing much in the plugin/repo yet. So, we may not want to use it.